Calling/App.xaml.cs

```diff
@@ -86,5 +147,65 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
             //TODO: Save application state and stop any background activity
             deferral.Complete();
         }
+
+        private async Task InitNotificationsAsync()
```

Comment:
I wonder if we should add some comments and links here telling Contoso how to mint package identifier and onboard to Azure notification to avoid the confusion that this code would work out of box.

Reply:
Added a comment with the tutorial link to get the values neeed.